pkgs/applications/editors/emacs/elisp-packages/manual-packages/sunrise-commander/default.nix

```
@@ -1,24 +1,43 @@
 { lib
-, trivialBuild
-, fetchFromGitHub
 , emacs
+, fetchFromGitHub
+, melpaBuild
+, writeText
 }:
 
-trivialBuild {
+let
   pname = "sunrise-commander";
-  version = "unstable=2021-09-27";
+  ename = "sunrise";
+  version = "20210924.1620";
 
   src = fetchFromGitHub {
     owner = "sunrise-commander";
     repo = "sunrise-commander";
     rev = "16e6df7e86c7a383fb4400fae94af32baf9cb24e";
     hash = "sha256-D36qiRi5OTZrBtJ/bD/javAWizZ8NLlC/YP4rdLCSsw=";
   };
+in
+melpaBuild {
+  inherit pname ename version src;
+
+  commit = src.rev;
 
-  buildInputs = [
+  outputs = [ "out" "doc" ];
+
+  packageRequires = [
     emacs
   ];
 
+  recipe = writeText "recipe" ''
+    (sunrise
```

This needs to be the same as `ename` (`ename` defaults to `pname`).

Is this better?

```suggestion
    (${ename}
```

---

I have a better idea: a `mkMelpaRecipe` function!